screens/ProductionScreens/ProductionDashboard.jsx

```diff
@@ -99,6 +105,21 @@ export default function ProductionDashboardScreen({ route }) {
                                 </Text>
                             );
                         })}
+
+                        <View className="flex flex-col m-2">
+                            <JoinViewBudgetButton
+                                text="Create Budget"
+                                onPress={() => setModal(<CreateBudgetModal />)}
+                            >
+                                <FontAwesomeIcon icon={faPiggyBank} size={50} />
```

Comment:
think I remember you saying it doesn't work, but can we just pass an icon in the props?

Reply:
So I think it would be possible, but a lot of example stuff I've seen for stuff suggests this way. Seems a tad cleaner than the dynamic imports idk